takwimu/takwimu_ui/src/Theme.js

```diff
@@ -1,6 +1,6 @@
 import { createMuiTheme } from '@material-ui/core/styles';
 
-const FONT_FAMILY_HEADING = '"Lora", serif';
+const FONT_FAMILY_HEADING = '"Lora", "serif"';
```

Comment:
No need for quotes around `serif` me thinks.

Reply:
perhaps